src/components/svg/keys/svg-keyboard-key/svg-keyboard-key.component.ts

```diff
@@ -68,7 +69,11 @@ export class SvgKeyboardKeyComponent implements OnInit, OnChanges, OnDestroy {
     private subscription: Subscription;
     private animation: string = 'inactive';
 
-    constructor(private mapper: MapperService, private store: Store<AppState>) {
+    @HostListener('click') onClick() {
```

Comment:
I remember something that we talked about this, but is don't find it elegant now. Maybe we should talk about it again and remind me what was reason.